packages/fxa-content-server/app/styles/modules/_settings.scss

```diff
@@ -42,6 +42,21 @@ body.settings #stage .settings {
   }
 }
 
+.cc-expired-alert {
+  position: absolute;
+  top: 0;
+  width: 100%;
+  background: #d70022;
+  color: white;
+  padding: 5px;
+  text-align: center;
+}
+
+.cc-expired-alert a {
+  text-decoration: underline;
+  color: white;
+}
```

Comment:
You can nest this selector.

Reply:
Yeah, might read a bit more easily if you put the `a {...}` block above nested in the `.cc-expired-alert` block.